unit-tests/src/test/java/org/eclipse/collections/impl/list/fixed/AbstractMemoryEfficientMutableListTestCase.java

```diff
@@ -61,6 +62,31 @@ public void testGetClass()
         Verify.assertInstanceOf(this.getListType(), this.list);
     }
 
+    @Test
+    public void replaceAll()
+    {
+        MutableList<String> emptyStrings = this.classUnderTest();
+        this.list.replaceAll(s -> "");
+        Assert.assertEquals(
+                Lists.mutable.withNValues(this.getSize(), () -> ""),
+                this.list);
+    }
+
+    @Test
+    public void sort()
+    {
+        this.list.shuffleThis();
+        this.list.sort(Comparator.naturalOrder());
+        Assert.assertEquals(
+                this.getNStrings(),
+                this.list);
+        this.list.shuffleThis();
+        this.list.sort(null);
+        Assert.assertEquals(
+                this.getNStrings(),
+                this.list);
+    }
+
     @Test
     public void sortThis()
     {
```

Comment:
Is sortThis() missing coverage for the override that takes a comparator?

Reply:
I'm looking at overall coverage for this entire hierarchy now.  Stay tuned.